src/agentplatform/adapter/rails.py

```
@@ -0,0 +1,37 @@
+
+
+class ToolRails:
+    def __init__(self, patterns: list[str] | tuple[str, ...]) -> None:
+        self._patterns = tuple(patterns)
+
+    @property
+    def patterns(self) -> tuple[str, ...]:
+        return self._patterns
+
+    def check(self, tool: str) -> bool:
+        """fail-closed：无匹配即拒绝。"""
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)